battstat/battstat-upower.c

```diff
@@ -83,9 +85,6 @@ battstat_upower_initialise (void (*callback) (void))
   int i, num;
 
   status_updated_callback = callback;
-#if UP_CHECK_VERSION (0, 99, 0)
```

Comment:
If anyone wonders, removing `#if` is fine as the actual application for this happens below.
Sharing the variable between functions would lead to eventual utilization of `up_client_get_devices` unconditionally anyway.